django/contrib/admin/options.py

```diff
@@ -1072,7 +1073,11 @@ def response_add(self, request, obj, post_url_continue=None):
                 'value': six.text_type(value),
                 'obj': six.text_type(obj),
             })
-            return SimpleTemplateResponse('admin/popup_response.html', {
+            return TemplateResponse(request, self.popup_response_template or [
+                "admin/%s/%s/popup_response.html" % (opts.app_label, opts.model_name),
+                "admin/%s/popup_response.html" % opts.app_label,
+                "admin/popup_response.html"
```

Comment:
Include a trailing comma in lists like these to allow adding more items later without having to modify this line again. Also, use single quotes rather than double. (We're using single quotes in new code except if the string contains a single quote.